src/modules/flow/utils/domParserUtils.ts

```
@@ -0,0 +1,44 @@
+/**
+ * DOM Parser utility that works in both browser and Node.js environments
+ */
+
+// Check if we're in a browser environment
+const isBrowser =
+  typeof window !== "undefined" && typeof window.DOMParser !== "undefined";
+
+// DOMParser implementation for Node.js
+let DOMParserImpl: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)